e2e_samples/temperature_events/LoadTesting/azure-pipeline.yml

```
@@ -0,0 +1,44 @@
+
+trigger:
+  - none
+
+pool:
+  vmImage: ubuntu-latest
+
+variables:
+  DeviceCount: 2
+  MessageCount: 5
+  Interval: 10
+  ContainerCount: 2
+  evhResourceGroup: 'temperature'
+  evhNamespace: 'temperature-dev'
+  evhName: 'temperature-processing'
+  serviceConnection: iotsimulator
+
+steps:
+# Store your eventhub connection string and subscription id in keyvault.
+- task: AzureKeyVault@1
+  inputs:
+    azureSubscription: $(serviceConnection)
+    KeyVaultName: temperature-keyvault
```

2 keyvaults globally can't have the same name.  This may need to be a variable so we can run it in different subscriptions.

```suggestion
    KeyVaultName: $(KeyVaultName)
```

---

thanks, fixed :)